.github/workflows/build.yml

```
@@ -11,6 +11,8 @@ jobs:
     steps:
       - name: Check out repo
         uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+      - name: Initialize ni-apis submodule
+        run: git submodule update --init ni-apis
```

The reasons why grpc-device uses a separate git command don't apply here, so let actions/checkout handle this.
```suggestion
      - name: Check out repo
        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
        with:
          submodules: true
```